lnprototest/runner.py

```diff
@@ -248,6 +248,17 @@ def close_channel(self, channel_id: str) -> None:
         """
         pass
 
+    @abstractmethod
+    def runner_features(
+        self,
+        additional_features: Optional[List[int]] = None,
+        fieldname: Optional[str] = None,
+    ) -> str:
+        """
+        Provide the features required by the node.
+        """
+        pass
```

Comment:
```suggestion
    def runner_features(
        self,
        additional_features: Optional[List[int]] = None,
        fieldname: Optional[str] = None,
    ) -> str:
        """
        Provide the features required by the node.
        """
        return ""
```

this can be the default one :)